Source/Utils/WarpXUtil.cpp

```diff
@@ -528,9 +538,19 @@ void ReadBCParams ()
                 WarpX::particle_boundary_lo[idim] = ParticleBoundaryType::Periodic;
                 WarpX::particle_boundary_hi[idim] = ParticleBoundaryType::Periodic;
             }
-
+        }
+        if (maxwell_solver_id == MaxwellSolverAlgo::PSATD) {
+            if (WarpX::field_boundary_lo[idim] == FieldBoundaryType::PEC ||
+                WarpX::field_boundary_hi[idim] == FieldBoundaryType::PEC) {
+                amrex::Abort(" PEC boundary not implemented for PSATD, yet!")
+            }
```

Comment:
@dpgrote @EZoni @RemiLehe As discussed offline, since PEC does not work with PSATD to give a clean standing wave, I have added this abort if PEC is used for PSATD